src/Marille/TopicConfiguration.cs

```diff
@@ -27,4 +27,11 @@ public struct TopicConfiguration () {
 	/// Max number of retries that will be performed when a worker fails to consume a message.
 	/// </summary>
 	public uint? MaxRetries { get; set; } = null;
+	
+	/// <summary>
+	/// Max number of parallel workers that will be able to consume messages from the channel. The default is to
+	/// queue all workers that need a background thread using the ThreadPool. This might result in overloading the
+	/// pool. If this property is set, the hub will ensure that we do not exceed a certain number of workers. 
+	/// </summary>
+	public uint? MaxParallelism { get; set; } = null;
```

Comment:
Shouldn't this be set to a reasonable default to put user into a secure by default state?

Reply:
The default would be that of the dotnet thread pool, which is calculated by the runtime to match the CPUs etc..